src/services/app-versions-service.ts

```diff
@@ -51,3 +51,28 @@ export const defaultVersionByAppId = async (appId: AppId, useLiveVersion = false
 
   return validVersion;
 };
+
+export const getAppVersionById = async (appVersionId: AppVersionId): Promise<AppVersion> => {
+  try {
+    const path = getAppVersionsByAppIdUrl(appVersionId);
+    const url = appsUrlBuilder(path);
+    logger.debug(`fetching logs url: ${url}`);
```

Comment:
wrong log

Reply:
Why? it's for --verbose
we already have something like this
@maorb-dev